lib/helper/core_oauth.dart

```diff
@@ -19,6 +19,8 @@ class CoreOAuth {
   Future<void> logout() async =>
       throw UnsupportedFailure(ErrorType.Unsupported, 'Unsupported logout');
 
+  Future<bool> get isLogged async => false;
```

Comment:
Will need to override in the mock oauth class to return true as that class pretends you are always logged in.